source/simulator/core.cc

```diff
@@ -1365,6 +1403,95 @@ namespace aspect
 
 
   template <int dim>
+  class VectorFunctionFromVectorFunctionObject : public Function<dim>
```

Comment:
It would be nice to have some more documentation what this class is for (as in VectorFunctionFromVelocityFunctionObject in the same file)